src/webgpu/api/validation/buffer/map_state.spec.ts

```diff
@@ -0,0 +1,8 @@
+export const description = `
+Validation tests for GPUBuffer.mapAsync, GPUBuffer.unmap and GPUBuffer.getMappedRange.
```

Comment:
These are already done in mapping.spec.ts

Reply:
oops, accidentally duplicated